itest/test/spectre/spectre_test.py

```diff
@@ -275,6 +275,7 @@ def test_post_always_cached_for_extended_json_content_type(self):
         assert response.status_code == 200
         assert response.headers['Spectre-Cache-Status'] == 'miss'
 
+        time.sleep(0.2)
```

Comment:
Use https://github.com/Yelp/casper/blob/master/itest/test/util.py#L38 with a method param or create a new similar one

Reply:
Yes that is much better !